py/strato/monitors/exceptionfinder.py

```diff
@@ -0,0 +1,213 @@
+import json
+from strato.common.log import config as logconfig
+import os
+import datetime
+import re
+import calendar
+import yaml
+import logging
+import time
+from collections import defaultdict
+
+FILEBEAT_CONFIG_FILE_PATH = 'tools/monitoring/filebeat.yml'
+OUTPUT_PATH = os.path.join(logconfig.LOGS_DIRECTORY, 'exceptions.stratolog')
+LOGS_WITH_COMMON_FORMAT = ['consul', 'nova', 'neutron', 'keystone']
+LOGS_DISTRIBUTION = defaultdict(int)
+LOGS_TYPES_FOR_ADJUSTMENT = list(LOGS_WITH_COMMON_FORMAT) + ['vm-console', 'mancala-bin']
+
+
+def start(hosts):
```

Comment:
as i mentioned in the PR message, cleanup part is still missing.
the point of it was to get the logic of the strato-log out.